ui/pages/settings/networks-tab/networks-list-item/networks-list-item.js

```diff
@@ -95,13 +97,13 @@ const NetworksListItem = ({
           <UrlIcon
             className="networks-tab__content__icon-with-fallback"
             fallbackClassName="networks-tab__content__icon-with-fallback"
-            name={label}
+            name={label || getNetworkLabelKey(labelKey)}
           />
         )
       )}
-      {network.isATestNetwork && (
+      {network.isATestNetwork && network.chainId !== CHAIN_IDS.LINEA_GOERLI && (
```

Comment:
Why was this added condition needed here?

Reply:
It is a temporay condition because linea mainnet network use the linea goerli rpc url and chainId